pkgs/applications/editors/edit/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchgit, unzip, pkgconfig, ncurses, x11, cwebbin }:
```

Comment:
I suppose more specific is better. In the end, you'll only end up bloating the build environment, so it's not that big of a deal (unless there are optional features you want to disable by omitting their dependencies from the environment).